### PR TITLE
Suggestions for the Intro

### PR DESCRIPTION
**Pull request by ohsOllila**
Here are suggestions to the intro. I suggest quite a bit of changes but I think that the content is still approximately the same, except that I have commented out the paragraph starting "By analyzing a wide ...", and I have move the paragraph starting "We analyze lipid dynamics ..." to the methods section.

Also the "Theoretical background" section is now titled "Evaluation of conformational dynamics of lipids against NMR data" and moved under methods as a subsection. I was planning to work on that section next.

Let me know what do you think about the suggestions for the Intro?

### CODE REVIEW

**Comment by ohsOllila** [comment]
There is now another commit "Suggestions for the Evaluation of conformational" which contains suggestions for the section "Evaluation of conformational dynamics of lipids against NMR data", which was formerly titled "Theoretical background".

If you want merge the commits partially or invidually, reject the pull request, and let me know what you want and I make a new one.

**Comment by ohsOllila** [comment]
There is now a third commit with comments on 'Simulational data acquisition and analysis' section. 

Same as before, if you want merge the commits partially or invidually, reject the pull request, and let me know what you want and I make a new one.

**Comment by markussmiettinen** [comment]
> Here are suggestions to the intro. I suggest quite a bit of changes but I think that the content is still approximately the same, except that I have commented out the paragraph starting "By analyzing a wide ...", and I have move the paragraph starting "We analyze lipid dynamics ..." to the methods section.
> 
> Let me know what do you think about the suggestions for the Intro?

I like the suggested Intro, and feel that it is considerably improved compared to the original.

The first two paragraphs were particularly nice. The third would benefit from some tuning, as would the fourth also to some extent.

But on the whole, I would accept this version of the Intro and continue the edits from that.

File: manuscripts/teff.tex [hunk 1]
```diff
@@ -76,59 +76,87 @@ \section{Introduction}
...
-To extract reliable statistics, it is crucial to assess that the simulations have converged: The conformations sampled have to represent the equilibrium distribution with enough transitions between states. Indeed, for lipids even 500\,ns simulations might be insufficient~\cite{vogel12,ferreira15}.
+%In addition to experimental results, the push from %scientific journals and
+%funders towards
+The idea of public availability and conservation of data has recently extended also to molecular dynamics (MD) simulation trajectories of biomolecules, and discussion on how and by whom these databases for dynamic information would be set up is currently active~\cite{Hildebrand:2019a,Abraham:2019a,Gygli:2020a,Abriata:2020a,Hospital:2020a}.
+%
+\todo{Mention GPCRmd~\cite{Rodriguez-Espigares:2019a}? SAMULI: I think yes.}
+%Notably,
+
+Since 2013 the NRMlipids Project (\url{ nmrlipids.blogspot.fi}) has
+promoted a open collaboration approach,
```
Comment (markussmiettinen) [inline manuscripts/teff.tex]:
```suggestion
promoted an open collaboration approach,
```

File: manuscripts/teff.tex [hunk 2]
```diff
@@ -76,59 +76,87 @@ \section{Introduction}
...
+biological lipids composing cellular membranes, which cannot be descibed by coordinates of single structure in contrast to
+folded proteins or DNA strands.
+%, in their biologically relevant state as the core components of the cell's membranes, are intrinsically unstructured.
+Realistic MD simulations can provide conformational ensemble and dynamics of such molecules as well as
+enable studies of their biological functions in complex biomolecules assemblies.
+However, current MD simulation force fields largely fail to capture conformational ensembles of lipid headgroups and
+disordered proteins \cite{botan15,antila19,??}.
+Therefore, the quality of MD simulations in databanks and other applications must be carefully assessed against
+experimental data.
+For lipid bilayers, such evaluation is possible against NMR and scatterign data \cite{ollila16}.
```
Comment (markussmiettinen) [inline manuscripts/teff.tex]:
```suggestion
For lipid bilayers, such evaluation is possible against NMR and scattering data \cite{ollila16}.
```

File: manuscripts/teff.tex [hunk 3]
```diff
@@ -76,59 +76,87 @@ \section{Introduction}
...
+The idea of public availability and conservation of data has recently extended also to molecular dynamics (MD) simulation trajectories of biomolecules, and discussion on how and by whom these databases for dynamic information would be set up is currently active~\cite{Hildebrand:2019a,Abraham:2019a,Gygli:2020a,Abriata:2020a,Hospital:2020a}.
+%
+\todo{Mention GPCRmd~\cite{Rodriguez-Espigares:2019a}? SAMULI: I think yes.}
+%Notably,
+
+Since 2013 the NRMlipids Project (\url{ nmrlipids.blogspot.fi}) has
+promoted a open collaboration approach,
+where the whole research process, from the initial ideas and discussions to
+the analysis methods, data and publications, are publicly available all the time \cite{botan15}.
+While the main focus of the NMRlipids project has been in conformational ensembles
```
Comment (markussmiettinen) [inline manuscripts/teff.tex]:
```suggestion
While the main focus of the NMRlipids Project has been in conformational ensembles
```

File: manuscripts/teff.tex [hunk 4]
```diff
@@ -76,59 +76,87 @@ \section{Introduction}
...
+of lipid headgroups and ion binding to lipid membranes \cite{botan15,catte16,antila19},
+it has also accumulated a collection 
+%such a databank~\cite{Miettinen:2019c} for
+of atomistic MD simulations of lipid membranes containing hundreds of trajectories ({\tt zenodo.org/communities/nmrlipids}).
+This data is also partially indexed at \url{www.nmrlipids.fi}.
```
Comment (markussmiettinen) [inline manuscripts/teff.tex]:
```suggestion
These data are also partially indexed at \url{www.nmrlipids.fi}.
```

File: manuscripts/teff.tex [hunk 5]
```diff
@@ -76,59 +76,87 @@ \section{Introduction}
...
+%%bilayer with one or few lipid types serve as an important model system, that have been successfully used
+%used~\cite{lyubartsev11,chau07,ferreira13,botan15, ferreira15,miettinen19,XXX}, and
+%hold vast potential to decipher, e.g., molecular mechanisms behind anesthetics~\cite{chau07,XXX}, the effect of cholesterol on membrane structure~\cite{XXX,ferreira13}, and the functioning of membrane proteins~\cite{lindahl08} \todo{add more references?}.
+
+Here, we demonstrate how publicly available set of MD simulation data can be utilized to
+rapidly evaluate how fast individual lipid molecules sample their conformational ensemble
+against experimental data in different force fields with unprecedented extent. 
+%Using this freely available resource we demonstrate here, for the first time, the viability of creating new scientific knowledge solely through analysis of pre-existing, open access MD simulation data.
+MD simulations with correct lipid dynamics are desired, for example, for the interpretation of
+NMR or other experiments detecing molecular dynamics and to understand dynamics of biological processes where
```
Comment (markussmiettinen) [inline manuscripts/teff.tex]:
```suggestion
NMR or other experiments detecting molecular dynamics and to understand dynamics of biological processes where
```

**Pull request closed without merging**